apps/marketplace/components/Brief/BriefResponseForm.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Form } from 'react-redux-form'
+import PageAlert from '@gov.au/page-alerts'
+
+import { required, validEmail } from '../../components/validators'
+import ErrorBox from '../../components/shared/form/ErrorBox'
+import Textfield from '../../components/shared/form/Textfield'
+import FilesInput from '../../components/shared/form/FilesInput'
+import Textarea from '../shared/form/Textarea'
+
+const BriefResponseForm = props => {
+  const { model, brief, supplierCode, submitClicked, handleSubmit } = props
+  const { briefResponseSuccess } = brief
+
+  let hasFocused = false
+  const setFocus = e => {
+    if (!hasFocused) {
+      hasFocused = true
+      e.focus()
+    }
+  }
+
+  return (
+    <div className="row">
+      <div className="col-sm-push-2 col-sm-8 col-xs-12">
+        <article role="main">
+          {briefResponseSuccess
+            ? <PageAlert as="success">
```

Comment:
I can't get this to display - I can see `briefResponseSuccess` is set to true and the response is created but no success message displayed